provider/resource_site.go

```diff
@@ -155,7 +155,7 @@ func readSite(d *schema.ResourceData, m interface{}) error {
 		"access_key": primaryAgentKey.AccessKey,
 	})
 
-	return nil
+	return err
```

Comment:
you could instead `return d.Set(....`

Reply:
Thanks -- https://github.com/signalsciences/terraform-provider-sigsci/pull/106/commits/23f71ebc001aeda713da0bf8344aa93e9646bb14